cert.go

```diff
@@ -273,6 +322,20 @@ func (c *Certificate) Sign(privKey PrivateKey, digest EVP_MD) error {
 	return c.insecureSign(privKey, digest)
 }
 
+// Sign a certificate using a private key and a digest name.
```

Comment:
Shouldn't this comment read "Sign a certificate **request** using a private key and a digest name."

Reply:
That it should. Good catch, a lot of this is copied and pasted.